qcodes/instrument_drivers/tektronix/keithley_7510.py

```diff
@@ -1,10 +1,367 @@
-from typing import cast
+import numpy as np
+from typing import cast, Optional, List, Union, Any
 
-from qcodes import VisaInstrument, InstrumentChannel
-from qcodes.utils.validators import Enum, Numbers, Ints
+from qcodes import VisaInstrument, InstrumentChannel, ParameterWithSetpoints
+from qcodes.instrument.parameter import invert_val_mapping, Parameter, \
+    DelegateParameter, MultiParameter
+from qcodes.utils.validators import Enum, Numbers, Ints, Lists, Arrays
 from qcodes.utils.helpers import create_on_off_val_mapping
 
 
+class DataArray7510(MultiParameter):
+    """
+    Data class when user selected more than one field for data output.
+    """
+    value = None
+
+    def __init__(self,
+                 names: tuple,
+                 shapes: tuple,
+                 setpoints: tuple,
```

Comment:
Please write these out to match the super class 

```
                 names: Sequence[str],
                 shapes: Sequence[Sequence[int]],
                 setpoints: Optional[Sequence[Sequence]],
```

Reply:
fixed.